packages/schema-wrapper/test/mini_test.py

```diff
@@ -4,7 +4,7 @@
 import json
 from unittest import TestCase
 
-
+# Q to ask: how to import the dicts besides using os.path?
```

Comment:
I prefer https://docs.python.org/3/library/pathlib.html. But yeah, that should work.